libraries/USB/src/USBHIDKeyboard.h

```diff
@@ -63,6 +63,7 @@ typedef union {
 #define KEY_DOWN_ARROW  0xD9
 #define KEY_LEFT_ARROW  0xD8
 #define KEY_RIGHT_ARROW 0xD7
+#define KEY_MENU        0xFE
```

Comment:
@andreock I am not sure about this one. Can you please post any docs where you get this from? Thanks

Reply:
I got it from here: https://github.com/msommacal/duckuino/blob/master/parser.y#L47

Reply:
have you tested them?

Reply:
yes

Reply:
@P-R-O-C-H-Y  changes are OK with me